melanges-de-linstitut-dominicain-detudes-orientales-en.csl

```diff
@@ -0,0 +1,260 @@
+  </macro>
+  <macro name="volume">
+    <number variable="volume"/>
+  </macro>
+  <macro name="issue">
+    <number variable="issue"/>
+  </macro>
+  <macro name="publisher-place">
+    <choose>
+      <if match="any" variable="publisher-place">
+        <text variable="publisher-place"/>
+      </if>
+      <else>
+        <text term="no-place" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if match="any" variable="publisher">
+        <text variable="publisher"/>
+      </if>
+      <else>
+        <text term="no-publisher" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="volume-number">
+    <group delimiter=" ">
+      <number variable="number-of-volumes"/>
+      <label variable="volume" form="short"/>
+    </group>
+  </macro>
+  <macro name="issued">
+    <choose>
+      <if match="any" variable="issued">
+        <date date-parts="year" form="numeric" variable="issued"/>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="original-date">
+    <date date-parts="year" form="numeric" variable="original-date" prefix="[" suffix="]"/>
+  </macro>
+  <macro name="url">
+    <text variable="URL" prefix="‹" suffix="›"/>
+  </macro>
+  <macro name="accessed">
+    <group delimiter=" ">
+      <text term="accessed"/>
+      <date form="text" variable="accessed">
+        <date-part name="day"/>
+        <date-part name="month"/>
+        <date-part name="year"/>
+      </date>
+    </group>
+  </macro>
+  <macro name="editor">
+    <group>
+      <names variable="editor">
+        <name delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize="false"/>
+        <label form="short" prefix=" (" suffix=")"/>
+      </names>
+    </group>
+  </macro>
+  <macro name="page">
+    <group delimiter=" ">
+      <label plural="never" variable="page" form="short"/>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <macro name="edition">
+    <text variable="edition"/>
+  </macro>
+  <macro name="title-short">
+    <choose>
+      <if type="book" match="any">
+        <text variable="title" form="short" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title" form="short" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+  <citation>
+    <layout>
+      <group delimiter=", " suffix=".">
+        <choose>
+          <if type="webpage" match="none">
+            <text macro="author"/>
+          </if>
+        </choose>
+        <text macro="title-short" font-style="italic"/>
+        <choose>
+          <if type="webpage" match="none">
+            <text macro="locator"/>
+          </if>
+        </choose>
+      </group>
+    </layout>
+  </citation>
+  <bibliography>
+    <layout>
+      <group delimiter=", " suffix=".">
+        <choose>
+          <if type="chapter" match="any">
```

Comment:
```suggestion
          <if type="chapter entry entry-dictionary entry-encyclopedia" match="any">
```

Reply:
There are differences in the way chapters and dictionary/encyclopaedia entries are treated, such as the treatment of the _in_ or the container-title, so we should leave the treatment of these categories separate